services/ui-src/src/components/layout/Footer.tsx

```diff
@@ -105,57 +114,93 @@ const sx = {
     },
   },
   footerTopFlex: {
-    minH: "12.5rem",
-    justifyContent: "space-between",
-    alignItems: "flex-start",
-    paddingY: "2rem",
-    ".mobile &": {
-      flexDirection: "column",
-      alignItems: "center",
-    },
+    flexWrap: "wrap",
   },
-  footerTopLogoFlex: {
-    paddingRight: "1rem",
-    ".mobile &": {
-      minWidth: "200px",
-      paddingRight: 0,
-      justifyContent: "center",
+  footerTopLeftContainer: {
+    flex: "0 0 50%",
+    maxWidth: "100%",
+    minWidth: "100%",
+    marginBottom: "1em",
+    ".desktop &": {
+      marginBottom: 0,
+      maxWidth: "50%",
+      minWidth: "auto",
     },
   },
-  footerRightLogosFlex: {
-    alignItems: "flex-start",
+  footerTopRightContainer: {
+    flex: "0 0 50%",
+    maxWidth: "100%",
+    minWidth: "100%",
+    ".desktop &": {
+      minWidth: "auto",
+    },
   },
-  footerRightLogosTextFlex: {
+  footerTopRightTopFlex: {
+    justifyContent: "center",
     flexDirection: "column",
-    gap: "1.375rem",
-    ".mobile &": {
-      gap: "0",
+    alignItems: "center",
+    ".desktop &": {
+      justifyContent: "flex-end",
+      flexDirection: "row",
     },
   },
-  mcrLogo: {
-    maxWidth: "12.125rem",
-    ".mobile &": {
-      maxWidth: "10.125rem",
+  footerCMSBrandingLeft: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    maxWidth: "100%",
+    padding: "0 1rem",
+    minWidth: "100%",
+    width: "100%",
```

Comment:
Question: Seems unnecessary to set all these widths the same. Can probably just set `width` to 100 and then auto.

Reply:
Can do away with width I believe. Need minWidth and maxWidth though for how diff browsers can handle it. minWidth in particular though is the true one we could keep